autoload/lsp/util.vim

```diff
@@ -111,17 +111,21 @@ var resolvedUris = {}
 
 # Convert a Vim filename to an LSP URI (file://<absolute_path>)
 export def LspFileToUri(fname: string): string
-  if resolvedUris->has_key(fname)
-    return resolvedUris[fname]
+  var fname_full: string = fname->fnamemodify(':p')
+
+  if resolvedUris->has_key(fname_full)
+    return resolvedUris[fname_full]
   endif
 
-  var uri: string = fname->fnamemodify(':p')
+  var uri: string
```

Comment:
I would assign `fname_full` to `uri` here instead of in the else block, as it's so far away that it's hard to realize that `uri` have been assigned.